docs/content/s3.md

```diff
@@ -6040,3 +6041,121 @@ d) Delete this remote
 
 y/e/d> y
 ```
+### Zata Object Storage {#Zata}
+
+[Zata Object Storage](https://zata.ai/) provides a secure, S3-compatible cloud storage solution designed for scalability and performance, ideal for a variety of data storage needs.
+
+Run:
+go build -o rclone.exe
+
+The S3 compatible gateway is configured using `rclone.exe config` with a type of `s3` and a provider name of `Zata`. Below is an example run of the configurator.
+
+First run:
+
+```
+rclone.exe config
+```
+
+```
+This will guide you through an interactive setup process:
+No remotes found, make a new one?
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+
+Enter name for new remote.
+name>My zata storage
+
+Type of storage to configure.
+Choose a number from below, or type in your own value.
+XX / Amazon S3 Compliant Storage Providers including AWS,(included Zata also) ...
+\ (s3)
```

Comment:
I think the formatting has gone wrong here - there are supposed to be spaces before the \ so it lines up. This is the same in the other ones below which I won't comment on.